ansible/roles/app/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+# tasks file for app
+
+- name: Add unit file for Puma
+  ansible.builtin.copy:
+    src: files/monapp.service
+    dest: /etc/systemd/system/monapp.service
+  notify: Reload puma
+
+- name: Add config for DB connection
+  ansible.builtin.template:
+    src: templates/db_config.j2
```

Comment:
Here you can use `db_config.j2` without `template` directory prefix (like you do when template `src: mongod.conf.j2` )